packages/api/src/apps/api/app.ts

```diff
@@ -64,14 +64,36 @@ export default async (env: ProcessEnv) => {
     },
     erc20s: tables.erc20s.JsMap(),
     stats: {
-      usd: {
-        latest: {
-          tvm: empStats.JsMap("Latest Tvm"),
-          tvl: empStats.JsMap("Latest Tvl"),
+      emp: {
```

Comment:
stats information is broken out per contract type, with global state meant to be the combination of each type. global state will come in future PR though its stubbed out here.